copier.go

```diff
@@ -123,43 +123,123 @@ func TestCopier(t *testing.T, store types.Storager) {
 				})
 			})
 		})
+	})
+}
+
+func TestCopierWithDir(t *testing.T, store types.Storager) {
+	Convey("Given a basic Storager", t, func() {
+		c, ok := store.(types.Copier)
+		So(ok, ShouldBeTrue)
+
+		d, _ := store.(types.Direr)
```

Comment:
```suggestion
		d := store.(types.Direr)
```

It's OK to panic directly.